### PR TITLE
Adds FRONTEND - UHC Behavioral Health

### LINKED ISSUE #975: Enforce more type checking for column names in Breakdowns.ts
See todo:

Screen Shot 2021-06-03 at 12 11 16 PM

### LINKED ISSUE #1214: Suppress empty options in map card group dropdown
Currently, in UHC variables (COPD, Diabetes, and behavioral health conditions introduced in #1337 ) where Suicide has data in different age buckets than the other variables.

Solution to both of these issues is to only populate the demographic group dropdown with groups that actually have data

### PR DESCRIPTION
**Pull request by benhammondmusic**
### [Visit Preview Link](https://deploy-preview-1355--health-equity-tracker.netlify.app/)

- [x] revert `/tmp` json files and `.env` before merge
- [x] add subset drug misuse to data sources page, meta data 
- [x] include a brief disclaimer on the `Illicit Opioid` and `Non medical Opioid Use` as subsets of “Non-Medical Drug Use” 
- [x] opioid categories data types of larger `drug use` condition
- [x] confirm why `illicit opioid` is in downloadable CSV 2020 but isnt appearing on UHF tracker. visiting [https://www.americashealthrankings.org/explore/annual/measure/drug_use_illicit_opioids/state/ALL](https://www.americashealthrankings.org/explore/annual/measure/drug_use_illicit_opioids/state/ALL) does a redirect
- [x] update our source page for UHC
- [x] confirm frontend tests are up to date still
- [x] confirm if `sample size too small` is correct for gray geos
- [x] fix `suicide` highest/lowest calculations
- [x] update ALL meta / text info to reflect 2021 data 
- [x] confirm if we should be using 2021 data if available
- [x] confirm if `non-medical drug use` should be displayed as "in last year"
- [x] remove or refactor `.getUniqueFieldValues()` when seeking only populated demographic groups for the dropdown 
- [x] thorough spot checks to confirm data accuracy 
- [x] finalize wording for all labels, titles, etc
- [x] adjust wording for `MetricConfig`
- [x] Frequent Mental Distress
- [x] Depression
- [x] Suicide (Deaths)
- [x] Illicit Opioid Use
- [x] FIX: filter for correct UHC age buckets on 100k map group dropdown (not for the table chart since we want to show rows with no data to show their population info)
- [x] Alcohol Use (Excessive Drinking)

closes #1214

Closes #975 

This can be used for social media, blog posts, and general intro to adding UHC data on Behavioral Health. We’re on target for early Dec. Please add/edit: 
> “In support of the Satcher Health Leadership Institute’s commitment to expand the topics reported on the Health Equity Tracker, we’re excited to announce the creation of a “Behavioral Health” category and the addition of 7 indicators including:

• Frequent Mental Distress
• Depression
• Suicide (Deaths)
• Illicit Opioid Use
• Non-Medical Drug Use
• Excessive Drinking

 > Significant disparities in outcomes, gaps in data, and other data quality issues exists for Behavioral Health, which makes it difficult or impossible to understand the impact of these issues among patient populations. Reducing behavioral health disparities, improving reporting, categorization, and standardization of definitions used between data sources and ultimately achieving health equity, requires understanding the wide range of factors that contribute to these outcomes. The Health Equity Tracker is built to empower analysis and comparison for these indicators to support programs, policies, and other efforts that advance Health Equity. To learn more about our sources and methodology, please click here.”

### CODE REVIEW

**Comment by benhammondmusic** [comment]
Screen Shot 2021-12-04 at 2 43 17 PM
Screen Shot 2021-12-04 at 2 43 24 PM

**Comment by benhammondmusic** [comment]
COMPLETED SPOT CHECKS:

## Depression
https://www.americashealthrankings.org/explore/annual/measure/Depression_a/state/ALL?edition-year=2020
- [x] U.S. Value: 19.9% === 19,900 cases of depression per 100k
- [x] Healthiest State: New Jersey: 11.8% === lowest rate: New Jersey: 11,800 per 100k
- [x] Least-healthy State: West Virginia: 28.8% === highest rate: West Virginia: 28,800 per 100k

> WHO IS AFFECTED?
>Populations of adults who have a higher prevalence of depression include: 
>Females compared with males.
TRUE
>Hispanic, white and Black adults compared with Asian adults. 
TRUE

## Excessive Drinking 
https://www.americashealthrankings.org/explore/annual/measure/ExcessDrink/state/ALL?edition-year=2020
- [x] U.S. Value: 18.6% === 18,600 /100k
- [x] Healthiest State: Utah: 12.0% ===  Lowest Rate Utah: 12,000 per 100k
- [x] Least-healthy State: Wisconsin: 23.6% === 2nd Highest Rate (after DC) Wisconsin: 23,600 per 100k

> WHO IS AFFECTED?
> The prevalence of excessive drinking is higher among:
> Males than females.
TRUE: 22,700 per 100k vs 14,100 per 100k
> Adults ages 18-44 compared with adults ages 45 and older. 
TRUE: 25,200 per 100k vs 15,600 per 100k
> Hawaiian/Pacific Islander, Hispanic and white adults compared with Asian adults.
TRUE


## Frequent Mental Distress
https://www.americashealthrankings.org/explore/annual/measure/mental_distress/state/ALL?edition-year=2020
- [ ] U.S. Value: 13.8% === 13,800 frequent mental distress cases per 100k in the United States
- [ ] Healthiest State: South Dakota: 10.6% === Lowest Rate: South Dakota: 10,600 per 100k
- [ ] Least-healthy State: West Virginia: 20.6% === Highest rate: West Virginia: 20,600 per 100k

> WHO IS AFFECTED?
> The prevalence of frequent mental distress is higher among:
> Females than males.
TRUE
> Adults ages 18-44 than older adults.
TRUE
> Multiracial and American Indian/Alaska Native adults than Black, white, Hispanic, Hawaiian/Pacific Islander and Asian adults.
TRUE

**Comment by benhammondmusic** [comment]
[9:13 AM] Jha, Madhuri
NISTA

[9:14 AM] Jha, Madhuri
NSSP

[9:15 AM] Jha, Madhuri
AHRQ

**Comment by benhammondmusic** [comment]
Emailed AHR re: the illicit opioid use being in the CSV but not on the website. 
MEDIA CONTACTS  
Jenifer McCormick, UnitedHealth Group
Office: 952-936-1917
jenifer_mccormick@uhg.com

**Comment by benhammondmusic** [comment]
Screen Shot 2022-01-06 at 4 58 52 PM

**Review by joshzarrabi — APPROVED** [review]

File: frontend/.env.deploy_preview [hunk 1]
```diff
@@ -1,4 +1,4 @@
 REACT_APP_DEPLOY_CONTEXT=deploy_preview
 REACT_APP_BASE_API_URL=https://dev.healthequitytracker.org
 
-REACT_APP_FORCE_STATIC=geographies.json
+REACT_APP_FORCE_STATIC=geographies.json,uhc_data-age.json,uhc_data-race_and_ethnicity.json,uhc_data-sex.json,acs_population-by_age_state.json
```
[the remark this answers is not in the document]
Comment (joshzarrabi) [reply frontend/.env.deploy_preview:4]: @benhammondmusic

**Pull request merged**